spec/defines/filter/client_auth_n_spec.rb

```diff
@@ -190,8 +190,9 @@
           'pass' => 'password',
           'uri'  => 'http://uri'
         },
-        :client_maps => [ 'testing', ],
-        :delegating  => 'true'
+        :client_maps         => [ 'testing', ],
+        :delegating          => 'true',
+        :delegating_quality  => '0.9'
```

Comment:
you mean I can't use '0.9' as a default value?  Does that mean I'd have to default the value in the class?  I.e.g, if undef, use '0.9'?